.github/workflows/go.yml

```diff
@@ -6,40 +6,38 @@ on:
       - main
       - release-*
     paths-ignore:
-      - 'docs/**'
-      - 'examples/**'
-      - '**.md'
+      - "docs/**"
+      - "examples/**"
+      - "**.md"
```

Comment:
I think we can remove all the quotes, except in lines 11, 13, 24 and 26, so that the auto-formatters have one thing less to bikeshed about.